libs/client/LightEpoch.cs

```diff
@@ -15,22 +15,22 @@ namespace Garnet.client
     public sealed unsafe class LightEpoch
     {
         /// Size of cache line in bytes
-        public const int kCacheLineBytes = 64;
+        private const int KCacheLineBytes = 64;
```

Comment:
I believe the lowercase ``k`` came from C++ convention for constants. I suggest this project to use normal PascalCase as per the [.NET Framework Design Guidelines guides:](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions)
```suggestion
        private const int CacheLineBytes = 64;
```

Reply:
Hi @PaulusParssinen , how are you?
I think that the access modifiers should be modified in many places, and what I saw in the code was not given much attention.
In changing the name of the variable, I tried not to change the original name and only change the naming convention
But if you agree, it can be corrected

Reply:
I'm good, thanks for asking 😄 

I agree with access modifiers being sort of all over the place in the current state of project. In many places the explicit access modifier is missing completely which makes it even worse, but state of this can be improved in later PRs by introducing good convention that are enforced with `.editorconfig` rules. 

As for this change, other files in the project use proper the naming convention for constant fields as I suggested above.

Reply:
Please remove the `K` prefix from the other constants below too 😄